app/components/Map.js

```diff
@@ -14,8 +16,48 @@ export default class Map extends Component {
       position: null,
       selectedPin: undefined,
       dropPinLocation: undefined,
+      loaded: false,
+      friendLocs: {},
     };
   }
+// Friend
+  componentWillMount() {
+    const { friends } = this.props;
+    let self = this;
+    let counter = 0;  
+     for(var friendId in friends) {
+      self.setListener(friends[friendId]);
+      counter++;
+      if(counter === Object.keys(friends).length) {
+        this.setState({loaded: true});
+      }
+    }
+  }
+
+  setListener(friend) {
+    let self = this;
+    currLoc.child(friend.id).on("value", function(snap) {
+      console.log('jake is moving')
+      self.state.friendLocs[friend.id] = snap.val();
+    });
+  }
+
+  renderFriends() {
+    // renders friends current locations
+    const { friends } = this.props;
+    let copy = this.state.friendLocs;
+    return _.map(copy, (coords, id) => {
+        return (
+        <MapView.Marker
+          coordinate={coords}
+          key={id}
+          image={{uri: friends[id].picture}}
```

Comment:
friends image seems small, especially compared to the pins now. Can we enlarge them?